scRNA-seq/directory_structure_comprehensive.txt

```diff
@@ -0,0 +1,109 @@
+├── 00-scRNA_introduction.Rmd
+├── 00-scRNA_introduction.html
+├── 01-scRNA_quant_qc-live.Rmd
+├── 02-filtering_scRNA-live.Rmd
+├── 03-normalizing_scRNA-live.Rmd
+├── 04-dimension_reduction_scRNA-live.Rmd
+├── 05-clustering_markers_scRNA-live.Rmd
+├── 06-celltype_annotation-live.Rmd
+├── README.md
+├── data
+│   ├── PBMC-TotalSeqB
+│   │   └── normalized
+│   │       └── PBMC_TotalSeqB_normalized_sce.rds
+│   ├── glioblastoma-10x
+│   │   └── normalized
+│   │       └── glioblastoma_normalized_sce.rds
+│   ├── glioblastoma-darmanis
+│   │   └── preprocessed
+│   │       ├── darmanis_metadata.tsv
+│   │       ├── qc_reports
+│   │       │   ├── fastp
+│   │       │   │   ├── SRR3934349_fastp.html
+│   │       │   │   ├── SRR3934349_fastp.json
+│   │       │   │   ├── SRR3934350_fastp.html
+│   │       │   │   ├── SRR3934350_fastp.json
+│   │       │   │   ...
+│   │       │   └── fastqc
+│   │       │       ├── SRR3934349
+│   │       │       ├── SRR3934350
+│   │       │       ...
+│   │       │       ├── fastqc_report_all.csv
+│   │       │       └── fastqc_report_filtered.csv
+│   │       ├── salmon_quant
+│   │       │   ├── SRR3934349
+│   │       │   │   ├── aux_info
+│   │       │   │   ├── cmd_info.json
+│   │       │   │   ├── libParams
+│   │       │   │   ├── lib_format_counts.json
+│   │       │   │   ├── logs
+│   │       │   │   └── quant.sf
+│   │       │   ├── SRR3934350
+│   │       │   │   ├── aux_info
+│   │       │   │   ├── cmd_info.json
+│   │       │   │   ├── libParams
+│   │       │   │   ├── lib_format_counts.json
+│   │       │   │   ├── logs
+│   │       │   │   └── quant.sf
+│   │       │   ...
+│   │       ├── sample_list.csv
+│   │       └── txi
+│   │           ├── count_matrix.tsv
+│   │           └── tximport_obj.RDS
```

Comment:
We don't use the darmanis data anymore, so it can be removed from these files completely, and we will want to make sure it is not included in the links that we create: we should remove those lines from `scripts/link-data.sh` and `scripts/syncup-s3.sh`.

Reply:
> I don't know if you used tree -l for the comprehensive file, but if you didn't you can!

I did `tree -l` and hoooooo boy was the output overwhelming so I just did it manually...!

> We don't use the darmanis data anymore

Excellent point! Since that's a bit of a separate task I opened https://github.com/AlexsLemonade/training-modules/issues/719